ldi-core/geojson-to-wkt/pom.xml

```diff
@@ -10,7 +10,9 @@
     </parent>
 
     <artifactId>geojson-to-wkt</artifactId>
-
+    <properties>
+        <jena-geosparql.version>4.7.0</jena-geosparql.version>
```

Comment:
do we need these properties? Since we are using ${jena.version} everywhere instead